.ci/pipeline_definitions

```diff
@@ -42,12 +42,12 @@ terraformer:
             target: terraformer
             build_args:
               PROVIDER: openstack
-          terraformer-packet:
+          terraformer-equinixmetal:
```

Comment:
FYI: the build for this PR will probably fail now because of the renaming, as the pipeline configs on the source branch are not considered for overwriting config on the base branch.
We will merge it nevertheless, on master it should build then.